src/main/java/com/tiph/dysonsphereproject/mixins/OverworldSkyMixin.java

```diff
@@ -0,0 +1,31 @@
+package com.tiph.dysonsphereproject.mixins;
+
+import net.minecraft.client.multiplayer.ClientLevel;
+import net.minecraft.client.renderer.DimensionSpecialEffects;
+import org.joml.Vector3f;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(DimensionSpecialEffects.OverworldEffects.class)
+public abstract class OverworldSkyMixin extends DimensionSpecialEffects {
+
+  public OverworldSkyMixin(
+      float p_108866_, boolean p_108867_, SkyType p_108868_, boolean p_108869_, boolean p_108870_) {
+    super(p_108866_, p_108867_, p_108868_, p_108869_, p_108870_);
+  }
+
+  @Override
+  public void adjustLightmapColors(
+      ClientLevel level,
+      float partialTicks,
+      float skyDarken,
+      float blockLightRedFlicker,
+      float skyLight,
+      int pixelX,
+      int pixelY,
+      Vector3f colors) {
+    colors = new Vector3f(255);
+
+    //        super.adjustLightmapColors(level, partialTicks, skyDarken, blockLightRedFlicker,
```

Comment:
Since this isn't really doing anything rn add like a TODO or comment about why its in the code base or just remove it, otherwise lg2m